### ISSUE TITLE
functional approach fails 

### ISSUE THREAD

**Issue by UGuntupalli** [sandialabs/pecos#61]
_****Update: I noticed that the Pip version is 0.1.7 while the stable version in GitHub and ReadTheDocs is 0.1.8** **_

The functional approach as documented in the "Framework" of the documentation shows the following: 

```python 
results = pecos.monitoring.check_range(data, [-3,3])
``` 

Here is my attempt at doing the exact same thing in multiple different ways: 
** ATTEMPT 1 **
```python 
# Import required packages 
import numpy as np
import pandas as pd
import re
from pandas.util import testing as tm
import pecos

# Set Seed and generate a dataframe for testing 
tm.N, tm.K = 15, 3
np.random.seed(256)
ts_df = tm.makeTimeDataFrame(freq='D')

# Reset Index
ts_df = ts_df.reset_index()

# Call check_range as shown in the documentation 
check_range_test = pecos.monitoring.check_range(ts_df,[-1,1])
```

It generates the following exception: 
`
module 'pecos.monitoring' has no attribute 'check_range' 
` 

** ATTEMPT 2 **
```python 
# Import required packages 
import numpy as np
import pandas as pd
import re
from pandas.util import testing as tm
from pecos import monitoring 

# Set Seed and generate a dataframe for testing 
tm.N, tm.K = 15, 3
np.random.seed(256)
ts_df = tm.makeTimeDataFrame(freq='D')

# Reset Index
ts_df = ts_df.reset_index()

# Call check_range as shown in the documentation 
check_range_test = monitoring.check_range(ts_df,[-1,1])
```

It generates the following exception: 
`
module 'pecos.monitoring' has no attribute 'check_range' 
` 

Can you kindly point out to what I am doing wrong ? Additionally, do you have any further documentation on the functional approach beyond what is on the ReadTheDocs ?

**Comment by kaklise** [comment]
Sorry for the confusion.  The functional approach is available in 0.1.8, which is available on GitHub.  We'll tag this version soon and push that out to PyPI.  Right now, the documentation follows the master branch on GitHub.

**Comment by UGuntupalli** [comment]
Katherine,
                 Hope you are doing well, but since the GitHub issue has been closed, I just checked the PyPi, but it doesn’t show that the updated package has been pushed through. Will you kindly let me know when you push it through ?
                 As we build the package more, we will be delighted to share our work with you ☺

[cid:image001.jpg@01D5B90D.A5B40710]

[cid:image002.png@01D5B90D.A5B40710]
[cid:image003.png@01D5B90D.A5B40710]
[cid:image004.png@01D5B90D.A5B40710]
Uday Guntupalli | Analytics Engineer III
First Solar, Inc. | 350 W. Washington St, Suite 600, Tempe, AZ 85281 USA
M: 1.480.231.9704 | www.Firstsolar.com<http://www.Firstsolar.com>
IMPORTANT: This email including any attachment may contain confidential information. If you received this message in error, please contact the sender and delete it from your computer. Any review, transmission, dissemination, or other use or taking of any action in reliance upon this message by persons or entities other than the intended recipient is prohibited and may be unlawful.
